### DISCUSSION TITLE
Question about a value

### DISCUSSION THREAD

**Discussion by jslegers73** [nicolargo/glances#2615] [Q&A]
Hello,

![image](https://github.com/nicolargo/glances/assets/6212673/c070ab59-9347-496a-a9e2-db292d1033c8)

I have a question what does the 8 mean?

**Comment by nicolargo** [comment]
What(s the result of the following command line ?

    glances --stdout quicklook.cpu_name

**Reply by jslegers73** [reply]
Hello,

I get the same: 
pi@rpi-test:~ $ glances --stdout quicklook.cpu_name
quicklook.cpu_name: 8
quicklook.cpu_name: 8
quicklook.cpu_name: 8

Maybe you can readout this value for a raspberry pi: 

pi@rpi-test:~ $ cat /sys/firmware/devicetree/base/model
Raspberry Pi 4 Model B Rev 1.2
pi@rpi-test:~ $

**Comment by nicolargo** [comment]
The algo used is the following: https://github.com/nicolargo/glances/blob/develop/glances/cpu_percent.py#L75-L82

So it read the content of the /proc/cpuinfo file.

Can you copy/paste your /proc/cpuinfo file ?

**Reply by jslegers73** [reply]
This is the result.

pi@rpi-test:~ $ cat /proc/cpuinfo
processor       : 0
BogoMIPS        : 108.00
Features        : fp asimd evtstrm crc32 cpuid
CPU implementer : 0x41
CPU architecture: 8
CPU variant     : 0x0
CPU part        : 0xd08
CPU revision    : 3

processor       : 1
BogoMIPS        : 108.00
Features        : fp asimd evtstrm crc32 cpuid
CPU implementer : 0x41
CPU architecture: 8
CPU variant     : 0x0
CPU part        : 0xd08
CPU revision    : 3

processor       : 2
BogoMIPS        : 108.00
Features        : fp asimd evtstrm crc32 cpuid
CPU implementer : 0x41
CPU architecture: 8
CPU variant     : 0x0
CPU part        : 0xd08
CPU revision    : 3

processor       : 3
BogoMIPS        : 108.00
Features        : fp asimd evtstrm crc32 cpuid
CPU implementer : 0x41
CPU architecture: 8
CPU variant     : 0x0
CPU part        : 0xd08
CPU revision    : 3

Revision        : b03112
Serial          : 1000000008bee947
Model           : Raspberry Pi 4 Model B Rev 1.2

**Comment by nicolargo** [comment]
Copy... on Raspberry Pi, the line number 5 (of the /proc/cpuinfo) does not return the CPU name but the CPU architecture type... 

Perhaps another file contain the information. Can you have a look ?

PS: i transform the current discussion into an issue.

**Comment by nicolargo** [comment]
Issue ==> https://github.com/nicolargo/glances/issues/2616
